jadx-gui/src/main/resources/i18n/Messages_ko_KR.properties

```
@@ -23,31 +23,30 @@ menu.update_label=새 버전 %s 이(가) 존재합니다!
 file.open_action=파일 열기 ...
 file.add_files_action=파일 추가
 file.open_title=파일 열기
-#file.open_project=Open project
+file.open_project=프로젝트 열기
 file.new_project=새 프로젝트
 file.save_project=프로젝트 저장
 file.save_project_as=다른 이름으로 프로젝트 저장...
-#file.reload=Reload files
-#file.live_reload=Live reload
-#file.live_reload_desc=Auto reload files on changes
-#file.export_mappings_as=
+file.reload=파일 다시 로드
+file.live_reload=라이브 로드
+file.live_reload_desc=파일 내용 변경 시 자동으로 다시 로드
+file.export_mappings_as=다른 이름으로 매핑 내보내기...
 file.save_all=모두 저장
 file.export_gradle=Gradle 프로젝트로 저장
 file.save_all_msg=디컴파일된 소스를 저장할 디렉토리 선택
 file.exit=나가기
 
-#start_page.title=Start page
-#start_page.start=Start
-#start_page.recent=Recent projects
-
+start_page.title=페이지 시작
+start_page.start=시작
+start_page.recent=최근 프로젝트
```

Missing empty line failing test :sob:
```suggestion
start_page.recent=최근 프로젝트

```